draft-ietf-lisp-pubsub.xml

```diff
@@ -294,14 +294,14 @@ Rec +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 
           <t>If the Map-Server receives a subscription request for an EID-Prefix not present in the mapping database, it SHOULD follow the same logic described in Section 8.4 of <xref target="RFC9301"></xref> and create a temporary subscription state for the xTR-ID to the least-specific prefix that both matches the original query and does not match any EID-Prefix known to exist in the LISP-capable infrastructure. Alternatively, the Map-Server can instead determine that such subscription request fails, and send a Negative Map-Reply following Section 8.3 of <xref target="RFC9301"></xref>. In both cases, the TTL of the temporary subscription state or the Negative Map-Reply SHOULD be configurable, with a value of 15-minutes being RECOMMENDED. </t>
 
-          <t>The subscription state can also be created explicitly by configuration at the Map-Server (possible when a pre-shared security association exists, see <xref target="security"></xref>) using a variety of means that are out of scope. If at the time the explicit subscription is configured there is no nonce that can be used for the explicit subscription state (e.g., from a different subscription already established with the same xTR when a single nonce is kept per xTR-ID), then both the xTR and Map-Server MUST be configured with the initial nonce to use. It is RECOMMENDED to have a configuration option to enable (or disable) the xTR to accept publication information for EID-Prefixes it did not explicitly subscribe to. By default, the xTR is allowed to modify explicit subscription state following the procedures described in this section, however the Map-Server MAY disable this via configuration.</t>
+          <t>The subscription state can also be created explicitly by configuration at the Map-Server (possible when a pre-shared security association exists, see <xref target="security"></xref>) using a variety of means that are out of scope. If at the time the explicit subscription is configured there is no nonce that can be used for the explicit subscription state (e.g., from a different subscription already established with the same xTR when a single nonce is kept per xTR-ID), then both the xTR and Map-Server MUST be configured with the initial nonce to use. It is RECOMMENDED to have a configuration option to enable (or disable) the xTR to accept publication information for EID-Prefixes it did not subscribe to itself. By default, the xTR is allowed to modify explicit subscription state following the procedures described in this section, however the Map-Server MAY disable this via configuration. If the Map-Server disables this, whenever the xTR tries to modify the explicit subscription the Map-Server MUST return a Negative Map-Reply (Loc-Count = 0 and Loc-AFI = 0) with ACT bits set to 4 "Drop/Policy-Denied".</t>
```

Comment:
```suggestion
          <t>The subscription state can also be created explicitly by configuration at the Map-Server (possible when a pre-shared security association exists, see <xref target="security"></xref>) using a variety of means that are out of scope. If at the time the explicit subscription is configured there is no nonce that can be used for the explicit subscription state (e.g., from a different subscription already established with the same xTR when a single nonce is kept per xTR-ID), then both the xTR and Map-Server MUST be configured with the initial nonce to use. It is RECOMMENDED to have a configuration option to enable (or disable) the xTR to accept publication information for EID-Prefixes it did not explicitly subscribe to. By default, the xTR is allowed to modify explicit subscription state following the procedures described in this section, however the Map-Server MAY disable this via configuration. If the Map-Server is instructed to disable modifying explicit subscriptions by xTRs themselves, whenever an xTR tries to modify an explicit subscription the Map-Server SHOULD return a Negative Map-Reply (Loc-Count = 0 and Loc-AFI = 0) with ACT bits set to 4 "Drop/Policy-Denied". The Map-Server MAY silently ignore these requests if, for example, these requests are received within a short interval.</t>
```

Reply:
The last MUST is too strong. The MS can simply ignore these request under some conditions.